app/model/TelegramNotifier.php

```
@@ -0,0 +1,166 @@
+    {
+        $user = $this->userModel->findRow($userId);
+        if (!$user) {
+            return;
+        }
+
+        $challengeUserRows = $this->context->table('challenge_user')
+            ->where('user_id', $userId)
+            ->where('active != ?', 0)
+            ->fetchAll();
+
+        $today = new \DateTime();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '68', column '22'). (undefined)

(MissingImport)